wizards/openacademy_wizard.py

```diff
@@ -0,0 +1,30 @@
+from odoo import models, fields, api
+
+
+class Wizard(models.TransientModel):
+    _name = "openacademy.wizard"
+    _description = "Openacademy Wizard"
+
+    session_id = fields.Many2one("openacademy.session", required=True)
+
+    partner_ids = fields.Many2many(
+        comodel_name="res.partner",
+        relation="openacademy_wizard_partner_rel",
+        column1="wizard_id",
+        column2="partner_id",
+        string="Partners",
+    )
+
+    @api.model
+    def default_get(self, fields_list):
+        result = super().default_get(fields_list)
+        model = self.env.context.get("active_model")
+        if model == "openacademy.session":
+            session_id = self.env.context.get("active_id")
+            if session_id:
```

Comment:
Don't need two conditionals, just extract active model and ID before conditional and add two conditions

Reply:
This was not applied as asked. Now there's only one conditional with one condition and active ID is always added

Reply:
It is applied, in the code is:
````python
@api.model
    def default_get(self, fields_list):
        result = super().default_get(fields_list).
        model = self.env.context.get("active_model")
        if model == "openacademy.session":
            session_id = self.env.context.get("active_id").
            result.update({"session_id": session_id})
        return result
```
This update here is not reflected in this PR.

Reply:
In that code, you're updating session_id even if active_id is False

Reply:
This is not applied yet. Could you check the example code I provided, please?

As I mentioned, here you are updating default value even if active IDs are not present.

You should extract active ID(s) and active model. If active ID is not set or active model doesn't match, you do nothing.